pkg/ecdsa/tss/protocol_ready.go

```diff
@@ -86,6 +109,27 @@ func readyProtocol(
 
 	switch ctx.Err() {
 	case context.DeadlineExceeded:
+		for _, memberID := range group.groupMemberIDs {
+			memberAddress, err := memberIDToAddress(memberID, publicKeyToAddressFn)
+			if err != nil {
+				logger.Errorf(
+					"could not convert member ID to address for a member of "+
+						"keep [%v]: [%v]",
+					group.groupID,
+					err,
+				)
+				continue
+			}
+			if ok := readyMembers[memberAddress]; !ok {
```

Comment:
actually I think it should be enough to just: `if readyMembers[memberAddress] {...}`

Reply:
Heh, yeah... Fixed in https://github.com/keep-network/keep-ecdsa/pull/654/commits/3100c2313a1de3f284c7e1b30326bbf74290910a